<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QIoTestClass</class>
 <widget class="QMainWindow" name="QIoTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QIoTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1111</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>窗口功能区</string>
    </property>
    <widget class="QPushButton" name="pushButtonOpenFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFindpoint">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>找点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEdit">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>修改on/off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelfCheck">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>自检</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLockScreen">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>锁定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>系统设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="title">
     <string>测试窗口</string>
    </property>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:4px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPause">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:4px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStepTest">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:4px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>单步测</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>ok/ng</string>
     </property>
     <widget class="QLabel" name="labelResult">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>坐标</string>
     </property>
     <widget class="QLabel" name="labelCoordinateL">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelCoordinateR">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Arrow.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>针点</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>图号</string>
     </property>
     <widget class="QLabel" name="labelCategory">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="labelPinL">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelPinR">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>坐标查</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>220</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>针点查</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>761</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>测试档案</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>NG记录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:4px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>退出测</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>测试图号</string>
    </property>
    <widget class="QListWidget" name="listWidgetUp">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>231</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>420</y>
      <width>251</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>档案图号</string>
    </property>
    <widget class="QListWidget" name="listWidgetDown">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveUp">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>360</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>/\ 
|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMoveDown">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>360</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>|
\/</string>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_8</zorder>
   <zorder>pushButtonMoveUp</zorder>
   <zorder>pushButtonMoveDown</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qiotest.qrc"/>
 </resources>
 <connections/>
</ui>
